Backend/webblog/blog/models.py

```diff
@@ -6,3 +6,10 @@ class Blog(models.Model):
     description = models.TextField(max_length=1500)
     created = models.DateTimeField(auto_now_add=True)
     owner = models.ForeignKey('auth.User', related_name='blog', on_delete=models.CASCADE, null=True)
+
+
+class Comment(models.Model):
+    blog = models.ForeignKey(Blog, related_name='comment', on_delete=models.CASCADE)
+    description = models.TextField(max_length=1500)
+    created = models.DateTimeField(auto_now_add=True)
```

Comment:
use  `TimeStampedModel` from `django-extensions` (https://github.com/django-extensions/django-extensions)
`from django_extensions.db.models import TimeStampedModel`

Reply:
This is causing many problems and I can't implement it properly.